docs/modules/ROOT/pages/api/erc721.adoc

```diff
@@ -193,75 +193,75 @@ NOTE: Implementing xref:api/introspection.adoc#SRC5Component[SRC5Component] is a
 [.contract-index#ERC721Component-Embeddable-Impls]
 .Embeddable Implementations
 --
-.IERC721Impl
-* xref:#IERC721-balance_of[`++balance_of(self, account)++`]
-* xref:#IERC721-owner_of[`++owner_of(self, token_id)++`]
-* xref:#IERC721-safe_transfer_from[`++safe_transfer_from(self, from, to, token_id, data)++`]
-* xref:#IERC721-transfer_from[`++transfer_from(self, from, to, token_id)++`]
-* xref:#IERC721-approve[`++approve(self, to, token_id)++`]
-* xref:#IERC721-set_approval_for_all[`++set_approval_for_all(self, operator, approved)++`]
-* xref:#IERC721-get_approved[`++get_approved(self, token_id)++`]
-* xref:#IERC721-is_approved_for_all[`++is_approved_for_all(self, owner, operator)++`]
-
-.IERC721MetadataImpl
-* xref:#IERC721Metadata-name[`++name(self)++`]
-* xref:#IERC721Metadata-symbol[`++symbol(self)++`]
-* xref:#IERC721Metadata-token_uri[`++token_uri(self, token_id)++`]
+.ERC721Impl
+* xref:#ERC721Component-balance_of[`++balance_of(self, account)++`]
+* xref:#ERC721Component-owner_of[`++owner_of(self, token_id)++`]
+* xref:#ERC721Component-safe_transfer_from[`++safe_transfer_from(self, from, to, token_id, data)++`]
+* xref:#ERC721Component-transfer_from[`++transfer_from(self, from, to, token_id)++`]
+* xref:#ERC721Component-approve[`++approve(self, to, token_id)++`]
```

Comment:
There's an indirection level but no added but this IMO. We had a direct link to the interface section, even when the interface doesn't have the full description like requirements, etc... We should add the full explanation of the method instead of a link to the interface as we are doing below. I think that's the scope of another PR, though.

Reply:
Continuing this discussion on [this thread](https://github.com/OpenZeppelin/cairo-contracts/pull/920#discussion_r1508373452)

Reply:
understood. let's advance with your proposal but let's make sure we give it a proper thought and document the decision later on

Reply:
https://github.com/OpenZeppelin/cairo-contracts/issues/726#issuecomment-1973481994